hub/integrations/tests/test_pytorch.py

```diff
@@ -483,6 +495,33 @@ def test_pytorch_collate(local_ds, shuffle):
         np.testing.assert_array_equal(batch[1], np.array([2, 2, 2, 2]).reshape(4, 1))
 
 
+@requires_torch
+@pytest.mark.parametrize("shuffle", [True, False])
+def test_pytorch_transform_collate(local_ds, shuffle):
+    local_ds.create_tensor("a")
+    local_ds.create_tensor("b")
+    local_ds.create_tensor("c")
+    for _ in range(100):
+        local_ds.a.append(0 * np.ones((300, 300)))
+        local_ds.b.append(1 * np.ones((300, 300)))
+        local_ds.c.append(2 * np.ones((300, 300)))
+
+    ptds = local_ds.pytorch(
+        batch_size=4,
```

Comment:
I think we can create a global constant called BATCH_SIZE=4, and use it where we need it